src/components/VmDetails/cards/DisksCard/DiskImageEditor.js

```diff
@@ -296,7 +296,7 @@ class DiskImageEditor extends Component {
                   />
                 }
                 { !createMode && !isImage &&
-                  <div id={`${idPrefix}-size-readonly`} className={style['editor-field-read-only']}>
+                  <div id={`${idPrefix}-size-edit`} className={style['editor-field-read-only']}>
```

Comment:
Are you sure this is `edit`?

Reply:
Humm. I don't think that block is even needed anymore.  I'll remove it and leave just the `FormControl` in place.